docs/FreeBSD.md

```diff
@@ -41,72 +34,46 @@ minio-example    7872440   38 7872402     0%    /minio-example
 ```
 
 Verify if it is writable
-
 ```sh
 touch /minio-example/testfile
- ls -l /minio-example/testfile
+ls -l /minio-example/testfile
 -rw-r--r--  1 root  wheel  0 Apr 26 00:51 /minio-example/testfile
 ```
 
 Now you have successfully created a ZFS pool for further reading please refer to [ZFS Quickstart Guide](https://www.freebsd.org/doc/handbook/zfs-quickstart.html)
 
 However, this pool is not taking advantage of any ZFS features, so let's create a ZFS filesytem on this pool with compression enabled.  ZFS supports many compression algorithms: lzjb, gzip, zle, lz4.  LZ4 is often the most performant algorithm in terms of compression of data versus system overhead.
-
-
 ```sh
 zfs create minio-example/compressed-objects
 zfs set compression=lz4 minio-example/compressed-objects
 ```
 
 To keep monitoring your pool use
-
 ```sh
 zpool status
  pool: minio-example
 state: ONLINE
  scan: none requested
 config:
 
-	NAME        STATE     READ WRITE CKSUM
-	minio-example  ONLINE       0     0     0
-	  md0       ONLINE       0     0     0
+NAME        STATE     READ WRITE CKSUM
+minio-example  ONLINE       0     0     0
+md0       ONLINE       0     0     0
 
 errors: No known data errors
 ```
 
-#### Step 2.
-
-Now start minio server on the ``/minio-example/compressed-objects``, change the permissions such that this directory is accessibly by a normal user
+#### Start Minio service
 
 ```sh
-chown -R minio-user:minio-user /minio-example/compressed-objects
+chown -R minio:minio /minio-example/compressed-objects
 ```
 
-Now login as ``minio-user`` and start minio server.
-
 ```sh
-curl https://dl.minio.io/server/minio/release/freebsd-amd64/minio > minio
-chmod 755 minio
-./minio server /minio-example/compressed-objects
+pkg install minio
```

Comment:
Works flawlessly @harshavardhana. One small thing since minio is already running, when i did step 1 and then proceeded to step 2. I got the following: I wanted to also know how to stop minio on FreeBSD. Could we add that?
```
root@server2:~ # chown -R minio:minio /minio-example/compressed-objects
root@server2:~ # pkg install minio
Updating FreeBSD repository catalogue...
FreeBSD repository is up to date.
All repositories are up to date.
Checking integrity... done (0 conflicting)
The most recent version of packages are already installed
root@server2:~ # sysrc minio_disks=/minio-example/compressed-objects
minio_disks: /home/user/Photos -> /minio-example/compressed-objects
root@server2:~ # service minio start
minio already running?  (pid=959).
```